core/sessions/orm.go

```diff
@@ -152,7 +196,10 @@ func (o *orm) CreateSession(sr SessionRequest) (string, error) {
 	if len(uwas) == 0 {
 		lggr.Infof("No MFA for user. Creating Session")
 		session := NewSession()
-		_, err = o.db.Exec("INSERT INTO sessions (id, last_used, created_at) VALUES ($1, now(), now())", session.ID)
+		err := o.q.Transaction(func(tx pg.Queryer) error {
+			_, err = o.q.Exec("INSERT INTO sessions (id, email, last_used, created_at) VALUES ($1, $2, now(), now())", session.ID, user.Email)
```

Comment:
This transaction is redundant. Ought it to be extended to cover all SQL queries in the entire function?

Reply:
Ah derp, yeah it didn't even use `tx`. I moved to transaction to wrap entire function but as you mentioned, but we will then need to rework all the `o.db. *` calls so going revert for now and follow up